### DISCUSSION TITLE
Loosing all my commits by deleting my old email address

### DISCUSSION THREAD

**Discussion by sarparslan** [community/community#78933] [New to GitHub]
I deleted my old email address but I lose all my commits attached to that email. In order to fix  it and get my commits back, I added my old email address back and verified it. However still my commits haven't come back. Is there anyway to get my commits back?

**Comment by erickdc7** [comment]


**Reply by queenofcorgis** [reply]
Hi @sarparslan , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by a7medelbatal** [comment]
i don't understand you
